custom_components/tuya_local/devices/tuya_000004rime_weather_station.yaml

```diff
@@ -0,0 +1,674 @@
+            value: Cloudy, rainy and snowy
+            mask: 000000FF00000000000000000000000000
+      - id: 103
+        optional: true
+        type: base64
+        name: Payload
+  - entity: sensor
+    name: Local humidity
+    class: humidity
+    dps:
+      - id: 132
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 1 temperature
+    class: temperature
+    dps:
+      - id: 133
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 1 humidity
+    class: humidity
+    dps:
+      - id: 134
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 2 temperature
+    class: temperature
+    dps:
+      - id: 135
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 2 humidity
+    class: humidity
+    dps:
+      - id: 136
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 3 temperature
+    class: temperature
+    dps:
+      - id: 137
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 3 humidity
+    class: humidity
+    dps:
+      - id: 138
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: select
+    name: Time mode
+    icon: "mdi:clock-edit"
+    category: config
+    dps:
+      - id: 102
+        type: boolean
+        name: option
+        mapping:
+          - dps_val: true
+            value: 24h
+          - dps_val: false
+            value: 12h
+  - entity: switch
+    name: Alarm
+    icon: "mdi:alarm-light"
+    category: config
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+      - id: 110
+        type: string
+        name: Alarm state
+        mapping:
+          - dps_val: 0
+            value: Alarm
+          - dps_val: 1
+            value: Silent
+          - dps_val: 2
+            value: Enabled
+          - dps_val: 3
+            value: Disabled
+      - id: 109
+        optional: true
+        type: base64
+        name: Settings payload
+      - id: 117
+        optional: true
+        type: bitfield
+        name: Notifications payload
+  - entity: select
+    name: Display units
+    icon: "mdi:temperature-celsius"
+    category: config
+    dps:
+      - id: 105
+        type: string
+        name: option
+        mapping:
+          - dps_val: c
+            value: C
+          - dps_val: f
+            value: F
+  - entity: light
+    name: LCD brightness
+    icon: "mdi:brightness-7"
```

Comment:
mdi:television-ambient-light is another good icon for display brightness (not that this one is bad).